File: Python-basics/Conditional-Statements-Advanced_3/Lab/fruit_shop_11.py
product = input()
day = input()
quantity = int(input())

work_day = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday']
weekend = ['Saturday', 'Sunday']
result = 0

if day in work_day:
    if product == "banana":
        result = 2.5 * quantity
    elif product == "apple":
        result = 1.20 * quantity
    elif product == "orange":
        result = 0.85 * quantity
    elif product == "grapefruit":
        result = 1.45 * quantity
    elif product == "kiwi":
        result = 2.70 * quantity
    elif product == "pineapple":
        result = 5.5 * quantity
    elif product == "grapes":
        result = 3.85 * quantity

elif day in weekend:
    if product == "banana":
        result = 2.7 * quantity
    elif product == "apple":
        result = 1.25 * quantity
    elif product == "orange":
        result = 0.9 * quantity
    elif product == "grapefruit":
        result = 1.60 * quantity
    elif product == "kiwi":
        result = 3 * quantity
    elif product == "pineapple":
        result = 5.6 * quantity
    elif product == "grapes":
        result = 4.2 * quantity
    
print(f"{result:.2f}"")

File: Javascript/JS_basics/Conditional_Statements_Advanced_3_Lab/fruitShop_11.js
function shop(input) {
    let product = input[0];
    let day = input[1];
    let quantity = Number(input[2]);
    let result = 0;

    if (day == 'Saturday' || day == 'Sunday') {
        if (product == 'banana') {
            result = quantity * 2.7;
        }
        else if (product == 'apple') {
            result = 1.25 * quantity;
        }
        else if (product == 'orange') {
            result = quantity * 0.9;
        }
        else if (product == 'grapefruit') {
            result = quantity * 1.6;
        }
        else if (product == 'kiwi') {
            result = 3 * quantity;
        }
        else if (product == 'pineapple') {
            result = quantity * 5.6;
        }
        else if (product == 'grapes') {
            result = quantity * 4.2;
        }
        else {
            console.log('error');
        }
    }
    else if (day == 'Monday' || day == 'Tuesday' || day == 'Wednesday' || day == 'Thursday' || day == 'Friday') {
        if (product == 'banana') {
            result = quantity * 2.5;
        }
        else if (product == 'apple') {
            result = 1.2 * quantity;
        }
        else if (product == 'orange') {
            result = quantity * 0.85;
        }
        else if (product == 'grapefruit') {
            result = quantity * 1.45;
        }
        else if (product == 'kiwi') {
            result = 2.7 * quantity;
        }
        else if (product == 'pineapple') {
            result = quantity * 5.5;
        }
        else if (product == 'grapes') {
            result = quantity * 3.85;
        }
        else {
            console.log('error');
        }
    }
    else {
        console.log('error');
    }

    if (result != 0) {
        result = (result).toFixed(2);
        console.log(result);
    }
}